vsts.yml

```diff
@@ -80,4 +80,12 @@ steps:
     searchFolder: junit
   condition: and(always(), ne(variables['ELECTRON_RELEASE'], '1'))
 
+- task: kasunkodagoda.slack-notification.slack-notification-task.SlackNotification@3
+  displayName: Post Slack Notification
+  inputs:
+    SlackApiToken: '$(slack_token)'
+    Channel: '#bot-nightly-releases'
+    Message: '$(Build.DefinitionName)-$(Build.BuildNumber) finished with a $(Agent.JobStatus) status.'
+  condition: and(always(), eq(variables['Build.Reason'], 'Schedule'))
```

Comment:
why `and(always(), X)`? isn't that exactly equivalent to just `X`?

Reply:
@nornagon default is success, not always so for tasks that should always run you have to explicitly declare it